views/index.ejs

```diff
@@ -45,7 +45,7 @@
         <div class="search_form__logo"></div>
         <div id="react_menu__container"></div>
         <div class="search_form__wrapper empty">
-          <div class="search_form__return icon-arrow-left" onmousedown="clearSearch(event, true)"></div>
+          <div class="search_form__return icon-arrow-left"></div>
```

Comment:
This situation with a `div` with no event attached to it caused issues on some mobile devices where the click on this pseudo-button was difficult to trigger. I would suggest to use a proper `<button>` here.

Reply:
I suggest to do this in a separate PR. I'll try to refactor the search bar layout.

Reply:
@amatissart @G-Ray
I agree it should be a real `<button>`, but I just tested on the iPhone and I don't see the problem re-appearing here with the `<div>`.
There is however a problem related to the very weird active area of the element, which is very misleading. Most often I focused the input while my finger seemed on the `<div>`.
![Capture d’écran de 2020-12-04 16-58-13](https://user-images.githubusercontent.com/243653/101185314-02600f00-3652-11eb-8be5-e7e75cf3bbe3.png)

I tested with a real `<button>` to be sure, but there is no improvement on this behavior, which already exists in prod.
We should definitely improve this element layout, but it's ok for me to do it in a separate PR.